### PR TITLE
Adding gates to non-implemented windows tests

### PR DESCRIPTION
**Pull request by PoppySeedPlehzr**

### CODE REVIEW

File: osquery/tables/system/tests/system_tables_tests.cpp [hunk 1]
```diff
@@ -39,14 +42,17 @@ TEST_F(SystemsTablesTests, test_os_version) {
...
 }
+#endif
 
 TEST_F(SystemsTablesTests, test_process_info) {
   auto results = SQL("select * from osquery_info join processes using (pid)");
   ASSERT_EQ(results.rows().size(), 1U);
 
   // Make sure there is a valid UID and parent.
   EXPECT_EQ(results.rows()[0].count("uid"), 1U);
+#ifndef WIN32
```
Comment (theopolis) [inline osquery/tables/system/tests/system_tables_tests.cpp]: Can we use:
```cpp
if (!isPlatform(TYPE_WINDOWS)) {
  EXPECT_NE(results.rows()[0].at("uid"), "-1");
}
```

File: osquery/sql/tests/sqlite_util_tests.cpp [hunk 2]
```diff
@@ -121,8 +121,10 @@ TEST_F(SQLiteUtilTests, test_affected_tables) {
 
 TEST_F(SQLiteUtilTests, test_table_attributes_event_based) {
   auto sql_internal = SQLInternal("select * from process_events");
+#ifndef WIN32
```
Comment (theopolis) [inline osquery/sql/tests/sqlite_util_tests.cpp]: Same as below, prefer `isPlatform`.

File: osquery/tables/system/tests/system_tables_tests.cpp (hunk 1 above)
Comment (theopolis) [inline osquery/tables/system/tests/system_tables_tests.cpp]: Any reason why this doesn't work, we have the processes table right?

**Comment by osqueryer** [comment]
:no_good: The commit b5acdda4acc2e26cc33b1f3963a536bc0f042200 (Job results: [384](https://jenkins.osquery.io/job/osqueryPullRequestAudit/384/)) failed the code audit.

**Comment by osqueryer** [comment]
:no_good: The commit f08bf9e609269d28a25d9ce948fde966615f7f9c (Job results: [385](https://jenkins.osquery.io/job/osqueryPullRequestAudit/385/)) failed the code audit.

**Comment by PoppySeedPlehzr** [comment]
I think we'll also want a `etc_protocols` style table as well right?

**Comment by theopolis** [comment]
I was debating that, but I think it's possible so why not! Maybe we can deprecate the terms `etc_hosts` and `etc_protocols` (set the as aliases of `hosts` and `protocols`). There should be a way to get all the network protocol information from Windows, right?

**Comment by PoppySeedPlehzr** [comment]
I genuinely think so, but I (or someone) may need to really start tearing into this problem to see. If we can't get all of the _exact_ same information I think we can at least get real close.

**Pull request merged**